README.md

```diff
@@ -141,6 +141,7 @@ To ensure your website gets the greatest exposure, add it both to [Awesome Volto
 - [VHS Ehrenamtsportal](https://vhs-ehrenamtsportal.de) (Website to help volunteers that help refugees for the [German Adult Education Association](https://www.volkshochschule.de/), developed by [kitconcept GmbH](https://kitconcept.com), 2018)
 - [VisitModena](https://www.visitmodena.it/it) (Tourist website of the Municipality of Modena, developed by [RedTurtle](https://www.redturtle.it), 2020)
 - [WISE-Freshwater](https://water.europa.eu/freshwater) (WISE-Freshwater, the Freshwater Information System for Europe. Developed by [Eau de web](https://eaudeweb.ro) for the European Environmental Agency, 2021)
+- [York College / CUNY] (https://www.york.cuny.edu/) (York College is part of The City University of New York (CUNY), one of the largest university systems in the United States, developed in-house, 2023)
```

Comment:
```suggestion
- [York College / CUNY] (https://www.york.cuny.edu/) (York College is part of The City University of New York (CUNY), one of the largest university systems in the United States. Developed in-house, 2023)
```

This is not a good look. It appears to affect only the home page, though.

![Screenshot 2024-04-30 at 12 39 13 PM](https://github.com/plone/volto/assets/102112/4049ce97-5052-491d-a669-6ac04394dc16)

```console
GET https://www.york.cuny.edu/static/css/client.0cf6c3d5.css net::ERR_ABORTED 404 (Not Found)   www.york.cuny.edu/:2
GET https://www.york.cuny.edu/static/js/runtime~client.9713af2a.js net::ERR_ABORTED 404 (Not Found)    www.york.cuny.edu/:2
GET https://www.york.cuny.edu/static/js/client.980b59bf.js net::ERR_ABORTED 404 (Not Found)   www.york.cuny.edu/:2
```

Can you check your deployment and verify it loads the missing CSS and JS files, and ping us back?

Also consider adding this to https://github.com/collective/awesome-volto